plugins/node/opentelemetry-instrumentation-fastify/test/instrumentation.test.ts

```diff
@@ -330,12 +330,12 @@ describe('fastify', () => {
 
           // done was not yet called from the hook, so it should not end the span
           const preDoneSpans = getSpans().filter(
-            s => !s.attributes[AttributeNames.PLUGIN_NAME]
+            s => !s.attributes[AttributeNames.PLUGIN_NAME] || s.attributes[AttributeNames.PLUGIN_NAME] === 'fastify'
```

Comment:
To my understanding this test logic is a check of the side-effect that hook added used to not have a plugin name. It has the default plugin name now in the v4 version of fastify.

To aid going between Fastify V3 and V4 in the tests, I kept both conditionals for now.